velox/functions/lib/string/StringImpl.h

```diff
@@ -499,7 +506,12 @@ trimAscii(TOutString& output, const TInString& input, TShouldTrim shouldTrim) {
       curPos--;
     }
   }
-  output.setNoCopy(StringView(start, curPos - start + 1));
+  auto view = StringView(start, curPos - start + 1);
+  if constexpr (std::is_same_v<TOutString, StringView>) {
+    output = view;
```

Comment:
This will create a temp `StringView`, please call it `output = StringView(start, curPos - start + 1)`

Reply:
Done